security/README.md

```diff
@@ -0,0 +1,33 @@
+## Security Announcements
```

Comment:
this should probably live in SECURITY.md see https://help.github.com/en/articles/adding-a-security-policy-to-your-repository

Reply:
Ah, sounds good. Thanks!

Reply:
@philips As I read more from the link you mentioned and some other related references, I agree that SECURITY.md should be created for users to see the security policies from the GitHub security tab. However, we can leave the `README` as such and provide a link to it from `.github/SECURITY.md` I will update the changes for your review. Thanks!